src/Apps/Show/Components/ShowContextCard.tsx

```diff
@@ -43,121 +43,133 @@ const CARD_IMAGE_WIDTHS = [
+      context_module: ContextModule.presentingFair,
+      context_page_owner_type: contextPageOwnerType,
+      context_page_owner_id: contextPageOwnerId,
+      context_page_owner_slug: contextPageOwnerSlug,
+      destination_page_owner_type: OwnerType.fair,
+      destination_page_owner_id: fair.internalID,
+      destination_page_owner_slug: fair.slug,
+      type: "thumbnail",
     }
 
-    if (!fair) return null
+    tracking.trackEvent(payload)
+  }
+
+  if (!fair) return null
 
-    return (
-      <GridColumns>
-        {fair.isActive && (
-          <Column span={6}>
-            <Text variant="lg-display">Part of {fair.name}</Text>
-          </Column>
-        )}
+  return (
+    <GridColumns>
+      {fair.isActive && (
         <Column span={6}>
-          <StyledLink
-            to={fair.href}
-            textDecoration="none"
-            onClick={handleClick}
-          >
-            <FairCard fair={fair} />
-
-            <Spacer y={2} />
-            <Box>
-              <Text variant="xl">{fair.name}</Text>
-
-              <FairTiming fair={fair} />
-            </Box>
-          </StyledLink>
+          <Text variant="lg-display">Part of {fair.name}</Text>
         </Column>
-      </GridColumns>
-    )
-  }
+      )}
+      <Column span={6}>
+        <StyledLink to={fair.href} textDecoration="none" onClick={handleClick}>
+          <FairCard fair={fair} />
+
+          <Spacer y={2} />
+          <Box>
+            <Text variant="xl">{fair.name}</Text>
+
+            <FairTiming fair={fair} />
+          </Box>
+        </StyledLink>
+      </Column>
+    </GridColumns>
+  )
+}
 
-  const PartnerInfo = () => {
-    const partnerHref = partner?.href
-    const partnerName = partner?.name
-    const imageUrls = compact(
-      // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
-      partner?.artworksConnection?.edges?.map(({ node }) => node?.image?.url)
-    )
+const PartnerInfo: React.FC<{ partner: Props["show"]["partner"] }> = ({
+  partner,
+}) => {
+  console.log(partner)
+  const {
+    contextPageOwnerId,
+    contextPageOwnerSlug,
+    contextPageOwnerType,
+  } = useAnalyticsContext()
+
+  const tracking = useTracking()
+
+  const partnerHref = partner?.href
+  const partnerName = partner?.name
+  const imageUrls = compact(
+    partner?.artworksConnection?.edges?.map(({ node }) => node?.image?.url)
+  )
 
-    const images = imageUrls.map((url, i) => {
-      const imageWidth = CARD_IMAGE_WIDTHS[i]
-      return cropped(url, { width: imageWidth, height: imageWidth })
-    })
+  const images = imageUrls.map((url, i) => {
+    const imageWidth = CARD_IMAGE_WIDTHS[i]
+    return cropped(url as string, { width: imageWidth, height: imageWidth })
+  })
 
+  const locationNames = (() => {
     const locationNames = limitWithCount(
       // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
-      filterLocations(partner.locations),
+      filterLocations(partner?.locations),
```

Comment:
This was the bug

Reply:
The rest of the changes were to fix eslint errors (no react components nested in other react components)